ClientApp/src/app/interceptors/token.interceptor.ts

```diff
@@ -31,16 +31,16 @@ export class TokenInterceptor implements HttpInterceptor {
     }
 
     return next.handle(request).pipe(
-      catchError((err:any) => {
+      catchError((err) => {
         if(err instanceof HttpErrorResponse){
           if(err.status === 401 ){
             this.auth.signOut()
             this.router.navigate(['/login'])
             return throwError(() => new Error('token expired, login again')) // console.log's this. might be better way to implement instead of console.logging it
```

Comment:
Is there a way this will be displayed to the user?

Reply:
No not really, I don't really remember an exact reason for doing it, mostly just for consistency of styling